cdhweb/pages/context_processors.py

```diff
@@ -14,3 +16,7 @@ def page_intro(request):
     if page_intro:
         return {'page_intro': page_intro}
     return {}
+
+
+def wagtail_site(request):
+    return {'site': Site.find_for_request(request)}
```

Comment:
fyi there's also a top-level context processor in `cdhweb/context_processors.py` that sets `site` to the _django_ site object. not sure how or if that interacts with this

Reply:
Whoa, good catch. I can't remember what we use that for.. I wonder if we still need both, or if the wagtail site is compatible enough with the base django site to use it instead... Should I rename the new one to wagtail site? Also, what do you think about including it in the top level context processors instead of adding a new one?

Reply:
One thing I did see it used for was in the `robots.txt` like we were talking about this morning...presumably the wagtail site should work the same? I'm fine with just replacing the method in the top-level context processor to return the wagtail site. It seems like the only thing we'd need the django one for is its hostname, which the wagtail one has.